src/components/Campaign/CampaignHero/CampaignHero.tsx

```diff
@@ -40,7 +45,11 @@ export const CampaignHero: FC<CampaignHeroProps> = ({ campaign }) => {
   } = useCampaignDisplayData(campaign);
 
   const handleGoBack = () => {
-    router.back();
+    if (window.history.length > 1) {
+      router.back();
+    } else {
+      router.push(AppPaths.Missions);
+    }
   };
```

Comment:
This was needed, as if you enter on the page directly, when you go back, you exit the app